ClangSharpPInvokeGenerator/CursorWriter.cs

```diff
@@ -373,6 +383,38 @@ private bool BeginHandleDeclRefExpr(CXCursor cursor, CXCursor parent)
             }
         }
 
+        private bool BeginHandleDLLExport(CXCursor cursor, CXCursor parent)
+        {
+            Debug.Assert(cursor.Kind == CXCursorKind.CXCursor_DLLExport);
+
+            if (_attachedData.TryGetValue(parent, out var data) && (data is AttachedFunctionDeclData functionDeclData))
```

Comment:
A dll export probably means you are processing the header with the wrong define. It might be useful to expose a way to surface diagnostics from our own processing up to the user...